modules/collector-source/pkg/collector/collector.go

```diff
@@ -1234,7 +1281,9 @@ func NewNetZonePricePerGiBMetricCollector() *metric.MetricCollector {
 	return metric.NewMetricCollector(
 		metric.NetZonePricePerGiBID,
 		metric.KubecostNetworkZoneEgressCost,
-		[]string{},
+		[]string{
+			source.UIDLabel,
```

Comment:
I don't know if this needs UID because it is a general price for a type of network traffic